hardware/LSL_helper.py

```diff
@@ -0,0 +1,45 @@
+
+from collections import deque
+from common import debugPrint
+import time
+
+try:
+	from pylsl import StreamInlet, resolve_byprop
+except ImportError:
+	debugPrint("Unable to import pylsl which is required to communicate with OpenBCI")
```

Comment:
This won't throw an error. It will just print the debug message. Fine if its still WIP but not if you were expecting an error to be thrown

Reply:
I am not trying to throw an error